### PR TITLE
Updated to TestFairySDK 1.4.4 and added many methods e.g. pause, resume.

### PR DESCRIPTION
**Pull request by bjornm**
Hello Ari! 

Thanks for creating the TiTestFairy module! Here comes a pull request, in case you have the time to update your (original) branch. Changes:
- Updated the TestFairySDK to 1.4.4
- Exposed many more methods in the module, such as pause(), resume(), checkpoint() and setCorrelationId()
- Bumped version to 0.2.0
- Added very basic documentation

Cheers,
Björn

### CODE REVIEW

**Comment by hiroara** [comment]
Hello Björn. Thanks for your contribution!

I reviewed this PR, but found an implementation to crash.
I'll write the comment about it immediately, then please refer it.

And I updated the example codes to review this PR. This codes were placed on [PR-2](https://github.com/hiroara/TiTestFairy/tree/PR-2) brach.
Please see [the diff code](https://github.com/bjornm/TiTestFairy/compare/master...hiroara:PR-2) if you like.

Thanks!

File: Classes/ComArihiroTitestfairyModule.m [hunk 1]
```diff
@@ -48,5 +48,64 @@ -(void)begin:(id)apitoken
     return;
 }
 
+-(void)pushFeedbackController:(id)args
+{
+    [TestFairy pushFeedbackController];
```
Comment (hiroara) [inline Classes/ComArihiroTitestfairyModule.m:53]: I found a crash on this line.

Logs is here.

```
[ERROR] The application has crashed with an uncaught exception 'NSInternalInconsistencyException'.
[ERROR] Reason:
[ERROR] Only run on the main thread!
...
```

I think that it needs `ENSURE_UI_THREAD_0_ARGS` macro.
I tried to add the macro, then fixed it.

Comment (bjornm) [inline Classes/ComArihiroTitestfairyModule.m:53]: Good spotted! I missed testing that that method. Sounds like a good fix.

**Comment by bjornm** [comment]
Great! 

The example code looks good! Please push when you are done and I will use your version instead of my fork.

**Comment by hiroara** [comment]
OK, I merged [PR-2 branch](https://github.com/hiroara/TiTestFairy/tree/PR-2). So I'll close this PR.

Thank you again for your contribution!

**Comment by bjornm** [comment]
Excellent! Thanks.

**Pull request merged**